packages/types/src/lib/interfaces.ts

```diff
@@ -1778,7 +1828,9 @@ export interface SignerLike {
   getAddress: () => Promise<string>;
 }
 
-export interface GetPkpSessionSigs extends GetSessionSigsProps {
+export interface GetPkpSessionSigs
```

Comment:
Do we allow running Lit Action for PKPSessionSigs? I thought that was a separate LitActionSessionSigs?

Reply:
yea, we do allow Lit action for PkpSessionSigs. That's actually something we can add, not sure if this part of the scope though. We can definitely create an abstract function `getLitActionSessionSigs`